demos/pager/css/pager.css

```diff
@@ -35,43 +35,58 @@
 }
 
 .demo-pager-container thead {
+    background-color: #e6e7e8;
     color: #343499;
-    background-color: #E6E7E8;
 }
 
 .demo-pager-container td {
-    padding: .5em .2em;
+    padding: 0.5em 0.2em;
 }
 
-.demo-pager-head-select { width:5%; }
```

Comment:
Is there a rule that reports single line entries like this?  I see you've expanded lots of these, and yet there's one that is still there.